src/main/resources/templates/index.html

```diff
@@ -1,15 +1,174 @@
 <!DOCTYPE html>
-<html xmls:th="www.thymeleaf.org">
+<html xmlns:th="www.thymeleaf.org">
 <head>
-<meta charset="ISO-8859-1">
-<title>Hello</title>
+    <meta charset="ISO-8859-1">
+    <title>Hello</title>
 </head>
 <body>
 <img src="kanaAirlines.png" alt="Kana Airlines">
-<h2>Hello World!</h2>
+<h2>Book a flight!</h2>

```

Comment:
Nice menu implementation, the dynamic data returns accurate results. Tested to work locally.